### PR TITLE
Feat/network detection

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

File: frontend/containers/App/App.jsx [hunk 1]
```diff
@@ -38,12 +38,8 @@ class App extends Component {
     const {actions} = this.props
 
     APIBridge.registerProgressCallback(actions.registerNetworkCall)
+    ConnectionMonitor(2000).registerStatusChangeCallback(actions.setNetworkStatus)
 
-    // setInterval(() => {
```
Comment (eduardoboucas) [inline frontend/containers/App/App.jsx]: Is this going to be used?

Comment (mingard) [inline frontend/containers/App/App.jsx]: Is what going to be used?

Comment (eduardoboucas) [inline frontend/containers/App/App.jsx]: The block that is commented out.

Comment (mingard) [inline frontend/containers/App/App.jsx]: The deleted block? No, that's why I deleted it

Comment (eduardoboucas) [inline frontend/containers/App/App.jsx]: My brain stopped for a second there. Ignore.

Comment (mingard) [inline frontend/containers/App/App.jsx]: 🤕

**Pull request merged**